corehq/apps/reports/standard/deployments.py

```diff
@@ -318,22 +326,26 @@ def process_rows(self, users, fmt_for_export=False):
                 last_builds = reporting_metadata.get('last_builds')
                 if last_builds:
                     last_build = self.get_data_for_app(last_builds, self.selected_app_id)
+                if device and is_commcare_user:
+                    device_app_meta = self.get_data_for_app(device.get('app_meta'), self.selected_app_id)
             else:
                 last_sub = reporting_metadata.get('last_submission_for_user', {})
                 last_sync = reporting_metadata.get('last_sync_for_user', {})
                 last_build = reporting_metadata.get('last_build_for_user', {})
+                if is_commcare_user and last_build.get('app_id') and device and device.get('app_meta'):
```

Comment:
the way things are aligned, why do you need the check for commcare_user? for web user, there won't be any devices, which you are checking for either way, so checking for commcare user is redundant here.